etl/steps/data/garden/urbanization/2024-01-26/ghsl_degree_of_urbanisation.meta.yml

```diff
@@ -0,0 +1,206 @@
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Urbanization
+    processing_level: minor
+    description_key:
+    - |-
+      **The Degree of Urbanisation (DEGURBA)** is a  method designed for capturing the urban-rural continuum and facilitating international comparisons. Developed by six international organizations and endorsed by the UN Statistical Commission, it employs a two-level classification system.
+
+      The first level categorizes territories into three classes: 1) cities, 2) towns and suburbs, and 3) rural areas. This classification helps distinguish urban areas (cities plus towns and suburbs) from rural areas, emphasizing the distinct differences between towns, semi-dense areas, cities, and rural regions.
+
+      The second level adds granularity by further dividing towns and semi-dense areas into towns and suburban or peri-urban areas, and splitting rural areas into villages, dispersed rural areas, and mostly uninhabited areas.
+
+      The classification process involves two steps. Initially, all 1 km² grid cells are categorized into one of three types: urban centers (contiguous grid cells with a density of at least 1500 inhabitants per km² and a total population of 50,000 or more, defining a city), urban clusters (contiguous grid cells with a density of at least 300 inhabitants per km² and a total population of 5,000 or more, defining a town and suburb), and rural grid cells (defining rural areas). These grid cell types are then used to classify smaller administrative or statistical spatial units.
+
+      This method is used with a residential population grid derived from household point locations, typically from geo-coded censuses or registers. Over 30 countries have published such data, and others are collecting it. If point data is unavailable, a disaggregation grid is used, with accuracy dependent on the spatial unit size.
+
+      For future predictions (2025 and 2030), a special method is used to estimate how built-up areas (like cities and towns) have grown or changed. This method involves two main steps:
```

Comment:
It depends on what charts you want to make with this, but it might be worthwhile splitting the 2025 and 2030 data into different variables, which can be marked as projections on line charts. It seems like they are also constructedusing a different method, so it might be best to do this anyway, up to you though.

Reply:
yep, I agree actually! :) thanks!